addons/web/tests/test_session_info.py

```diff
@@ -25,7 +27,7 @@ def setUpClass(cls):
             tz="UTC")
         cls.user.write({
             'company_id': cls.company_a.id,
-            'company_ids': [Command.set([company.id for company in cls.companies])],
+            'company_ids': [Command.set([company.id for company in cls.allowed_companies])],
```

Comment:
Having a recordset enables such simplifications:
```suggestion
            'company_ids': [Command.set(cls.allowed_companies.ids)],
```